readme.md

```diff
@@ -46,6 +46,14 @@ Every time you change your `.toolkitrc.yml`, e.g. adding or removing a plugin, y
 npx dotcom-tool-kit --install
 ```
 
+### Updating existing Tool Kit configuration
+
+If your project already has a `.toolkitrc.yml` file you can add a new plugin to it by following the steps in the README for that plugin.
+
+At any time you can run `npx dotcom-tool-kit --help` to review the full list of hooks and tasks configured in your project.
+
+You can see the list of all available hooks and tasks provided by Tool Kit in the FAQs.
```

Comment:
I will link to the FAQs once that PR is merged (https://github.com/Financial-Times/dotcom-tool-kit/pull/334)